lib/bgs/student_school.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module BGS
+  class StudentSchool
+    def initialize(proc_id:, vnp_participant_id:, payload:, user:)
+      @user = user
+      @proc_id = proc_id
+      @vnp_participant_id = vnp_participant_id
+      @dependents_application = payload['dependents_application']
+    end
+
+    def create
+      bgs_service.create_child_school(submit_params(child_school))
+      bgs_service.create_child_student(submit_params(child_student))
```

Comment:
I would have expected that `BGS::StudentSchool.new.create` would return a "school", but as written it returns whatever `bgs_service.create_child_student` returns (presumably a "child student"? 🤷 ). Just want to make sure this is intentional.

Reply:
Good point @omgitsbillryan . We actually are not concerned with what gets returned, we have no need for it in subsequent calls. 

Perhaps `.call` is better?